### PR TITLE
Stripped node_modules and build sys of unneeded things!

### PR DESCRIPTION
**Pull request by kirkness**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,27 +32,8 @@
     "redux": "^3.6.0"
   },
   "scripts": {
-    "start": "node scripts/start.js",
+    "dev": "node scripts/start.js",
```
Comment (pchennz) [inline package.json]: should be rename the script to "dev" too for consistency?

File: src/core/relay-decorators.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+/**
+* Wrap a component in a relay container.
+*
+* Use this as a decorator.
+*/
+export function relayContainer(fragments) {
+  return function decorator(component) {
+    return Relay.createContainer(component, { fragments });
+  };
+}
```
Comment (pchennz) [inline src/core/relay-decorators.js]: we decided to hold off relay for now?

Comment (kirkness) [inline src/core/relay-decorators.js]: Oopppp, dont know how this got in there...

File: package.json (hunk 1 above)
Comment (kirkness) [inline package.json]: Cool yup

**Pull request closed without merging**